tizen/src/options.h

```diff
@@ -0,0 +1,61 @@
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_OPTIONS_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_OPTIONS_H_
+
+enum class ExistingWorkPolicy { replace, keep, append, update };
+
+enum class NetworkType {
+    connected,
+    metered,
+    not_required,
+    not_roaming,
+    unmetered,
+    temporarily_unmetered,
+};
+
+enum class OutOfQuotaPolicy {
+    run_as_non_expedited_work_request,
+    drop_work_request,
+};
```

Comment:
Observe the naming convention.
ex) `connected` -> `kConnected`, `not_required` -> `kNotRequired`

Reply:
Fixed it!